apps/src/fish/FishView.jsx

```diff
@@ -162,7 +162,6 @@ class FishView extends React.Component {
               width: Math.round(containerWidth),
               height: Math.round(containerHeight)
             }}
-            dir="ltr"
```

Comment:
can you provide some more context for what this was originally added to fix? It would probably also be worth looping in someone who was involved with that original fix to confirm that this change won't be a regression.

Reply:
Original commit: https://github.com/code-dot-org/code-dot-org/commit/bfa55f46c642044c3f3706daf9a1dfea92a23e6a

After speaking with Bethany, my understanding is that RTL rendering was breaking the robot animation in this activity. Because RTL was not yet supported in Oceans, LTR was forced to maintain correct animation.

This commit: https://github.com/code-dot-org/ml-activities/pull/352 begins RTL support by forcing LTR on only some animated elements in Oceans, including the robot and some buttons. Therefore, removing the LTR tag on the parent element doesn't affect the robot and button rendering.

Reply:
great, thanks! This change makes a ton of sense given that history